source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -1278,7 +1279,7 @@ export class CardDesigner extends Designer.DesignContext {
         root.style.flexDirection = "column";
         root.style.overflow = "hidden";
 
-        root.innerHTML =
+        setInnerHtml(root,
```

Comment:
as I noted in our chat, the designer is going to need much more extensive work than the base renderer, as all of this static HTML should be constructed through DOM APIs or some other mechanism. for now we should probably just revert the designer changes.